src/main/java/ch/njol/skript/classes/ClassInfo.java

```diff
@@ -277,6 +277,10 @@ public static boolean isValidCodeName(final String name) {
 		return this;
 	}
 	
+	public String[] getRequiredPlugins() {
+		return requiredPlugins;
+	}
+	
```

Comment:
This should probably be moved to go with the rest of the methods after the `getSince` method. It should also be marked as nullable.